extras/nginx-mtls-auth/Dockerfile.redhat

```diff
@@ -20,6 +20,7 @@ USER root
 RUN     set -e ; \
         set -x ; \
         mkdir /certs ; \
+        unset LD_LIBRARY_PATH ; \
```

Comment:
Why is it needed?

Reply:
after changes in openssl/crypto lib in ovms package and upgrading it, it clashes with  microdnf package installer so it fails to install systemd package.